front-end/src/components/Admin/index.js

```diff
@@ -89,6 +96,22 @@ class Admin extends Component {
 
     currentTab.contentUrl = url;
     return currentTab.contentUrl;
+    /* let currentTabIndex = this.state.tabs.findIndex((tab) => {
```

Comment:
You can delete this comment. I've saved a code snippet that does this state merge properly in this task:
https://trello.com/c/cKRQIDLt/196-remove-direct-state-mutating-in-admin-component
However updating state via setState breaks things here, so I don't think we will be doing that task in the nearest time.